### PR TITLE
Separate modules for server and middleware.

### PR DESCRIPTION
**Pull request by navilperez**
Create separate modules for the handlers that interact with the
database. events_handlers interact with the events table to enable the
core realtime collaboration function. users_handlers enable markers on
the workspace.

### CODE REVIEW

File: blockly-rtc/server/http/events_handlers.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ */
+async function queryEventsHandler_(res, serverId) {
+  try {
+    const entries = await database.query(serverId);
+    res.setHeader('Content-Type', 'application/json');
+    res.statusCode = 200;
+    res.write(JSON.stringify({ entries }));
+    res.end();
+  } catch {
+    console.log('why am i here');
```
Comment (samelhusseini) [inline blockly-rtc/server/http/events_handlers.js]: Is this intended?

File: blockly-rtc/server/http/events_handlers.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      res.statusCode = 200;
+      res.end();  
+    });
+  } catch {
+    res.statusCode = 401;
+    res.end();
+  };
+};
+
+module.exports.queryEventsHandler_ = queryEventsHandler_;
```
Comment (samelhusseini) [inline blockly-rtc/server/http/events_handlers.js]: Perhaps if it's being exported publicly it shouldn't have an ``_`` in the name? or at least the exported name shouldn't

File: blockly-rtc/server/http/events_handlers.js (hunk 1 above)
Comment (navilperez) [reply blockly-rtc/server/http/events_handlers.js]: Yikes. Leftover from debugging. Thanks for the catch.

File: blockly-rtc/server/http/events_handlers.js (hunk 2 above)
Comment (navilperez) [reply blockly-rtc/server/http/events_handlers.js]: Good point.

**Review by samelhusseini — APPROVED** [review]

**Pull request merged**